www/js/components/map-page.js

```diff
@@ -16,9 +16,21 @@ class MapPage extends Component {
 
   render() {
     const { marker, services, alerts } = this.props;
-    let selectedPoint = services[0];
-    if (marker.selectedMarker) {
+
+    let selectedPoint;
+    if(marker.selectedMarker) {
       selectedPoint = marker.selectedMarker;
+    } else if(services.length > 0) {
+      selectedPoint = services[0];
+    } else {
+      selectedPoint = { };
```

Comment:
Yeah, I don't know how the point card behaves with a null selectedPoint, since services[0] has always worked with our mock data.